src/sklearn.jl

```diff
@@ -408,6 +408,8 @@ function generate_gaussian_quantiles(; mean::Union{Array{<:Number, 1}, Nothing}
                                        shuffle::Bool = true,
                                        random_state::Union{Int, Nothing} = nothing)
 
+    typeof(mean) != Nothing && length(mean) != n_features && throw("length of mean must be equal to n_features.")
```

Comment:
Você pode, nesse caso, especificar o tipo de erro.

```throw(DomainError(x, "argument must be nonnegative"))```

Esse seria um ```ArgumentError```, não?

Reply:
Eu fiquei na dúvida, pensei que seria um `DimensionMismatch`, mas não especifiquei.

Reply:
ArgumentError: The parameters to a function call do not match a valid signature. Argument msg is a descriptive error string.

DimensionMismatch: The objects called do not have matching dimensionality. Optional argument msg is a descriptive error string.

Acho que pode ser o segundo